src/compiler/transformers/test/parse-props.spec.ts

```diff
@@ -34,6 +34,39 @@ describe('parse props', () => {
     expect(t.cmp?.hasProp).toBe(true);
   });
 
+  it('should correctly parse a prop with an unresolved type', () => {
+    const t = transpileModule(`
+    @Component({tag: 'cmp-a'})
+      export class CmpA {
+        @Prop() val?: Foo;
+      }
+    `);
+    expect(getStaticGetter(t.outputText, 'properties')).toEqual({
+      val: {
+        attribute: 'val',
+        complexType: {
+          references: {
+            Foo: {
+              id: 'global::Foo',
+              location: 'global',
```

Comment:
So TS thinks any type it can't resolve is just a global? Or is this something we're doing?

Reply:
yeah basically, if a type isn't imported into the module or declared inside of it we can safely assume it's a global type